lib/storefront/truck-removal.svg

```diff
@@ -0,0 +1,6 @@
+<svg data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 60 60">
+  <title>Delivery truck with removal icon</title>
+  <desc>Image of a delivery truck with a minus sign</desc>
+  <path d="M56.278 21.663a7.946 7.946 0 0 0-15.856-.763H25.06a.762.762 0 0 0-.763.762v2.122h-9.272a.763.763 0 0 0-.693.444l-3.242 7.08-5.83 1.223a.763.763 0 0 0-.605.747v8.316a.762.762 0 0 0 .762.762h6.218a4.032 4.032 0 0 0 4.036 3.633 4.086 4.086 0 0 0 4.035-3.633H35.92a4.054 4.054 0 0 0 3.994 3.23 4.115 4.115 0 0 0 3.99-3.23h7.426a.762.762 0 1 0 0-1.523h-1.865V29.52a7.95 7.95 0 0 0 6.813-7.857zM48.33 15.24a6.423 6.423 0 1 1-6.423 6.423 6.43 6.43 0 0 1 6.424-6.424zM6.18 33.9l5.6-1.176a.763.763 0 0 0 .535-.43l3.2-6.982H24.3v15.52h-4.762a4.06 4.06 0 0 0-7.734 0H6.18V33.9zm9.49 10.568A2.482 2.482 0 0 1 13.124 42a2.55 2.55 0 1 1 2.548 2.464zm24.244-.4a2.482 2.482 0 0 1-2.548-2.468 2.542 2.542 0 0 1 2.548-2.468 2.483 2.483 0 0 1 2.55 2.468 2.543 2.543 0 0 1-2.55 2.46zm3.994-3.23a4.055 4.055 0 0 0-3.994-3.238 4.115 4.115 0 0 0-3.99 3.23h-10.1V22.423h14.6a7.95 7.95 0 0 0 7.52 7.165V40.83h-4.036z" fill="#505150"/>
+  <path d="M45.54 22.424h5.58a.762.762 0 0 0 0-1.523h-5.58a.762.762 0 0 0 0 1.524z" fill="#505150"/>
```

Comment:
we usually remove the inline fills so we can add the color in css, can we do that here?